### PR TITLE
Update french translation of setup chapter

### PR DESCRIPTION
**Pull request by Krenodeno**

### CODE REVIEW

File: fr/02_Environnement_de_développement.md [hunk 1]
```diff
@@ -365,62 +321,28 @@ clean:
...
+Lancer `make test` doit vous afficher le programme sans erreur, listant le nombre d'extensions disponible pour Vulkan.
+L'application devrait retourner le code de retour 0 (succès) quand vous fermez la fenêtre vide.
 Vous devriez désormais avoir un makefile ressemblant à ceci :
 
 ```make
-VULKAN_SDK_PATH = /home/user/VulkanSDK/x.x.x.x/x86_64
-
-CFLAGS = -std=c++17 -I$(VULKAN_SDK_PATH)/include
-LDFLAGS = -L$(VULKAN_SDK_PATH)/lib `pkg-config --static --libs glfw3` -lvulkan
+CFLAGS = -std=c++17 01_Modules_shaders.md
```
Comment (abnercoimbre) [inline fr/02_Environnement_de_développement.md:329]: Did we actually mean to say:

`CFLAGS = -std=c++17 -O2`

?

File: fr/02_Environnement_de_développement.md [hunk 2]
```diff
@@ -263,6 +217,18 @@ Cette librairie contenue intégralement dans les headers peut être installée d
 sudo apt install libglm-dev
 ```
 
+### Compilateur de shader
+
+Nous avons tout ce qu'il nous faut, excepté un programme qui compile le code [GLSL](https://en.wikipedia.org/wiki/OpenGL_Shading_Language) lisible par un humain en bytecode.
+
+Deux compilateurs de shader populaires sont `glslangValidator` de Khronos et `glslc` de Google. Ce dernier a l'avantage d'être proche de GCC et Clang à l'usage,.
+Pour cette raison, nous l'utliliserons: téléchargez les exécutables [non officiels](https://github.com/google/shaderc/blob/main/downloads.md) et copiez `glslc` dans votre répertoire `/usr/local.bin`. Notez que vous aurez certainement besoin d'utiliser `sudo` en fonctions de vos permissions.
```
Comment (abnercoimbre) [inline fr/02_Environnement_de_développement.md:225]: Should be `/usr/local/bin`

File: fr/02_Environnement_de_développement.md (hunk 1 above)
Comment (Krenodeno) [reply fr/02_Environnement_de_développement.md:329]: Yep, I surely had a miss-paste and didn't see it. Thanks for the review !

Not sure if I can add a commit in this PR, or if I have to open another one :thinking:

Comment (Krenodeno) [reply fr/02_Environnement_de_développement.md:329]: I opened a new PR: #200

**Pull request merged**